client/src/annoMatrix/whereCache.ts

```diff
@@ -49,27 +49,38 @@ creates a cache entry of:
 }
 */
 import { _getColumnDimensionNames } from "./schema";
-import { _hashStringValues } from "./query";
+import { _hashStringValues, Query } from "./query";
+import { Field, Schema } from "../common/types/entities";
+
+export interface WhereCache {
+  summarize?: {
+    [key: string]: {
+      [key: string]: WhereCacheTerms;
+    };
+  };
+  where?: {
+    [key: string]: WhereCacheTerms;
+  };
+}
+
+interface WhereCacheTerms {
+  [key: string]: Map<string, Map<string, string[] | number[]>>;
+}
 
-// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types --- FIXME: disabled temporarily on migrate to TS.
 export function _whereCacheGet(
-  // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any -- - FIXME: disabled temporarily on migrate to TS.
-  whereCache: any,
-  // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any -- - FIXME: disabled temporarily on migrate to TS.
-  schema: any,
-  // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any -- - FIXME: disabled temporarily on migrate to TS.
-  field: any,
-  // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any -- - FIXME: disabled temporarily on migrate to TS.
-  query: any
-) {
+  whereCache: WhereCache,
+  schema: Schema,
+  field: Field,
+  query: Query
+): string[] | number[] | [undefined] {
```

Comment:
This is missing Int32Array

Suggestion:  define a type for "ColumnLabels" and use it both here and in WhereCacheTerms

Eg,
```
type WhereCacheColumnLabels = string[] | number[] | Int32Array;

interface WhereCacheTerms {
  [key: string]: Map<string, Map<string, WhereCacheColumnLabels>>;
}

// [snip]

export function _whereCacheGet(
  whereCache: WhereCache,
  schema: Schema,
  field: Field,
  query: Query
): WhereCacheColumnLabels | [undefined] {
  // ...
}
```

Reply:
As per related comment above, added/updated [whereCache.ts::WhereCacheColumnLabels](https://github.com/chanzuckerberg/cellxgene/blob/mim/annoMatrix/client/src/annoMatrix/whereCache.ts#L66).